browser/src/control/Control.Toolbar.js

```diff
@@ -210,6 +210,44 @@ function getConditionalFormatMenuElementsImpl(more, type, count, unoCommand, jsd
 	return table;
 }
 
+function sendUnoCommandWithSlideLayoutParam(num, unoCommand, jsdialogDropdown) {
+	var params = {
+		WhatLayout: {
+			type: 'long',
+			value: num,
+		}
+	};
+	map.sendUnoCommand(unoCommand, params);
+
+	if (jsdialogDropdown)
+		JSDialog.CloseAllDropdowns();
+	else
+		closePopup();
+}
+
+
+function getSlideLayoutPopup(type, unoCommand, jsdialogDropdown) {
```

Comment:
Also we should later avoid new 'htmlcontent' -> with jsdialog widgets we will get styling, a11y out of the box, so less work.